src/Components/test/E2ETest/Tests/WebAssemblyAuthenticationTests.cs

```diff
@@ -412,9 +466,16 @@ private void EnsureDatabaseCreated(IServiceProvider services)
 
         public void Dispose()
         {
-            // Make the tests run faster by navigating back to the home page when we are done
-            // If we don't, then the next test will reload the whole page before it starts
-            Browser.FindElement(By.LinkText("Home")).Click();
+            try
+            {
+                // Make the tests run faster by navigating back to the home page when we are done
+                // If we don't, then the next test will reload the whole page before it starts
+                Browser.FindElement(By.LinkText("Home")).Click();
+            }
+            catch
+            {
+                // We don't care if this step fails.
```

Comment:
How or why could that step fail?

Reply:
We don't need to do this anymore as we are using a different isolation context per test now, which ultimately results in a much better and stable experience. I've taken this out.

Reply:
removed